base/linalg/symmetric.jl

```diff
@@ -279,15 +280,15 @@ end
 (-)(A::Hermitian{Tv,S}) where {Tv,S} = Hermitian{Tv,S}(-A.data, A.uplo)
 
 ## Matvec
-A_mul_B!(y::StridedVector{T}, A::Symmetric{T,<:StridedMatrix}, x::StridedVector{T}) where {T<:BlasFloat} =
-    BLAS.symv!(A.uplo, one(T), A.data, x, zero(T), y)
+A_mul_B!(y::StridedVector{T}, A::RealHermSymComplexSym{T,<:StridedMatrix}, x::StridedVector{T}) where {T<:BlasReal} =
```

Comment:
This is wrong; `Symmetric{Complex}` will now end up in generic_matvec instead (since `T` has a different meaning in the `RealHermSymComplexSym` alias). Will push a correct version tomorrow.